src/main/features/claim/form/models/claimAmountBreakdown.ts

```diff
@@ -48,7 +43,17 @@ export default class ClaimAmountBreakdown implements Serializable<ClaimAmountBre
   }
 
   appendRow () {
-    this.rows.push(ClaimAmountRow.empty())
+    if (this.rows.length < MAX_NUMBER_OF_ROWS) {
```

Comment:
I would expect some sort of error when call is about to exceed limit. Is that business requirement that only up to 20 rows is allowed?